src/caselawclient/Client.py

```diff
@@ -213,7 +228,7 @@ def make_request(
         data: Optional[Dict[str, Any]] = None,
     ) -> requests.Response:
         kwargs = self.prepare_request_kwargs(method, path, body, data)
-        self.session.headers = headers  # type: ignore
```

Comment:
For some reason, this annotation is no longer valid in mypy on Github, but is mandatory locally.